### ISSUE TITLE
[content] Updates from the DACO team

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/dac-ui#569]
Please make the following content updates (still waiting on clarification for others, so this ticket may grow)

## Homepage

- [x] 1. The Overview paragraph, change the last sentence to: `The ICGC DACO is the overarching authority to ensure that data from the ICGC will only be used by qualified individuals for public health objectives. `

- [x] 2. For the third step of the application process, remove the link and change the wording to: 
`The ICGC DACO will review the application and approved project teams will be granted 2 years of access to ICGC Controlled Data.`

- [ ] 3. Waiting on one more change from Amber's email to Rita for the text in the banner

![image](https://user-images.githubusercontent.com/18424138/161807791-ebd5482a-10e7-4d4a-8301-526ce8f8f337.png)

## Dashboard

- [x] Remove the link for "eligible project teams" in this dashboard card
![image](https://user-images.githubusercontent.com/18424138/162001183-2a98f586-aa56-4f83-8c48-945cbb26d7f3.png)

## Application Terms section

- [x] In the last two sentences of this paragraph change the first one to the following, and remove the last sentence that has the link to the argo website.
` If your application is approved, you agree that your applicant's name, institution, and scientific lay summary may be included in a public registry of projects that have been granted access to ICGC Controlled Data.`
![image](https://user-images.githubusercontent.com/18424138/161818395-de413541-7021-40df-9d23-31ed3a6a2ed1.png)

## Project section D

- [x] Remove this full sentence, with the link to icgc argo site
![image](https://user-images.githubusercontent.com/18424138/162212846-4f3ee4a0-f737-477a-a133-21b61d1c68c2.png)

**Comment by ciaranschutte** [comment]
ok in dev

**Comment by kcullion** [comment]
All of these corrections have been made on QA. For any other content fixes, I will just make a new ticket. Thanks @ciaranschutte !

**Comment by ciaranschutte** [comment]
> All of these corrections have been made on QA. For any other content fixes, I will just make a new ticket. Thanks @ciaranschutte !

sounds good thank you
